src/style.css

```diff
@@ -1,3 +1,92 @@
-body {
+:root {
+  --primary-color: #00acdc;
+}
+
+body,
+div,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+ol,
+li,
+a,
+img,
+table,
+th,
+td {
   margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is loaded correctly in your HTML file. You have linked to Google Fonts in your HTML, so make sure the link is correct and the font is available.